python/src/trezorlib/cardano.py

```diff
@@ -415,8 +434,8 @@ def _parse_pool_owner(pool_owner) -> messages.CardanoPoolOwner:
     )
 
 
-def _parse_pool_relay(pool_relay) -> messages.CardanoPoolRelayParameters:
-    pool_relay_type = int(pool_relay["type"])
+def _parse_pool_relay(pool_relay: dict) -> messages.CardanoPoolRelayParameters:
+    pool_relay_type = messages.CardanoPoolRelayType(int(pool_relay["type"]))
```

Comment:
dtto, please try removing the inner `int()`

Reply:
Removed in b70acaf